.github/workflows/deploy.yaml

```diff
@@ -220,14 +220,17 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: setup_micromamba
-        uses: mamba-org/provision-with-micromamba@main
+        uses: mamba-org/setup-micromamba@v1
         with:
-          environment-file: False
           environment-name: mda
-          extra-specs: |
-            python==${{ matrix.python-version }}
+          create-args: >-
+            python=${{ matrix.python-version }}
             pip
-          channels: jaimergp/label/unsupported-cudatoolkit-shim, conda-forge, bioconda
+          condarc: |
+            channels:
+              - jaimergp/label/unsupported-cudatoolkit-shim
```

Comment:
I see it was already there, but worth documenting this? I had to go and search the following: https://github.com/openmm/openmm/issues/3059#issuecomment-891653746

Reply:
Sure. I guess I just run in circles where everyone knows what this shim is.